Build.PL

```diff
@@ -31,6 +31,7 @@ my $builder = Module::Build->new(
         'XML::Parser'        => 0,
         'XML::Parser::Style::IxTree' => 0,
         'XML::Twig'          => 0,
+        'XML::Tidy'          => 0,

```

Comment:
Let's not require a plugin-specific dependency at installation time. See how you can gracefully fail if the module is not present:
https://github.com/evernote/serge/blob/master/lib/Serge/Engine/Plugin/parse_csv.pm#L48-L49